src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -329,6 +330,29 @@ begin
            ennreal.tendsto_coe, ennreal.to_nnreal_coe],
 end
 
+lemma tendsto_lintegral_nn_of_le_const (μ : finite_measure α)
+  {fs : ℕ → (α →ᵇ ℝ≥0)} {c : ℝ≥0} (fs_le_const : ∀ n a, fs n a ≤ c) {f : α → ℝ≥0}
+  (fs_lim : ∀ a, tendsto (λ (n : ℕ), (fs n a)) at_top (𝓝 (f a))) :
+  tendsto (λ (n : ℕ), (∫⁻ a, (fs n a) ∂(μ : measure α))) at_top
+          (𝓝 (∫⁻ a, (f a) ∂(μ : measure α))) :=
+begin
+  simpa only using tendsto_lintegral_of_dominated_convergence (λ _, c)
```

Comment:
If you use `tendsto_lintegral_filter_of_dominated_convergence` instead of `tendsto_lintegral_of_dominated_convergence` then you can extend this lemma from the `at_top` filter over `nat` to any countably generated filter over a generic type. You can then hopefully do the same for all following results in this PR.

Reply:
I did this. I simultaneously generalized to almost everywhere assumptions.

I nevertheless also kept the original statements (now obtained as corollaries from the more general ones you suggested), since each requires three `eventually_of_forall`s, some inside `λ`s.

If the original ones are viewed as duplication, one can consider removing these special cases. But I think they are more convenient to use; in particular when working with bounded continuous functions, usually the bounds and convergence really hold everywhere and not just almost everywhere (and usually one really works with sequences).